codes/15686_py3_sjskgks.py

```diff
@@ -0,0 +1,35 @@
+from itertools import combinations
+import sys
+input = sys.stdin.readline
+
+sizeOfcity, min_ch = map(int, input().split()) # ch: chicken house
+posOfch = []
+posOfhouse = []
+for row in range(sizeOfcity):
+    col=0
+    for kind in map(int,input().split()):
+        if kind==1:
+            posOfhouse.append([row, col])
+        elif kind==2:
+            posOfch.append([row,col])
+        col+=1

```

Comment:
enumerate를 쓰면 더 간단해질 것 같습니다.
```suggestion
    for col, kind in enumerate(map(int,input().split())):
        if kind==1:
            posOfhouse.append([row, col])
        elif kind==2:
            posOfch.append([row,col])
```